### ISSUE TITLE
Multiple Issues With Text Editor in Firefox 65 vs. Chrome 72.0

### ISSUE THREAD

**Issue by skilfullycurled** [publiclab/plots2#4908]
Hi,

I've been experiencing a number of problems with the text editor.  Not sure if I should have split them up so apologies on that.  I've attached comparison images for every issue except issue #2.

The first two issues take place in Firefox 65 but not in Chrome 72.0 (macOS 10.12.6):

1) The last four buttons on the rich text editor are not functional in Firefox 65.  They can't be pressed, and there is no tool tip when you hover.

2) Although Firefox is autosaving (auto-saving?), Firefox does not display the fade-in/out "autosaving" indicator that Chrome does.  Sorry, no picture, it happens too fast.

The next three issues take place in _both_ Firefox 65 and Chrome 72.0.

3) When you scroll the page, the rich text editor buttons stay nicely in place as your content rolls behind it.  However, when you scroll to the bottom of the page, it reappears at the very bottom (image attached).

Now, take a look at the following [question](https://publiclab.org/questions/bsugar/03-02-2019/how-do-i-interpret-the-relationship-between-the-schema-rb-file-and-the-models) I posted:

4) Note the list, ( 1), 2a), 2b) ).  I made this list by typing a 1) then hitting return, then 2a) and so on.  This is what I intended.  However, when you go to edit the post, the rich text editor automatically re-formats the 1) as part of an indented list.

5) Additionally, you'll see below that the code formatting does not render correctly on the published note (you can also see a backtick), but it _is_ rendered correctly in the rich text editor.
![issue_1](https://user-images.githubusercontent.com/950291/53755784-65bd7b80-3e85-11e9-8ee2-96ecc696cd88.png)
![issue_3](https://user-images.githubusercontent.com/950291/53755785-65bd7b80-3e85-11e9-893e-dbf5069345f7.png)
![issue_4](https://user-images.githubusercontent.com/950291/53755786-65bd7b80-3e85-11e9-91c6-39aaf1e02554.png)
![issue_5](https://user-images.githubusercontent.com/950291/53755787-65bd7b80-3e85-11e9-9b5e-04404cb847be.png)

**Comment by jywarren** [comment]
thanks for the great documentation on this and for reporting the issues!

**Comment by skilfullycurled** [comment]
Np @jywarren.  N. P.

**Comment by skilfullycurled** [comment]
Updated so that "hash numeral 2" does not link to issue number two.

**Comment by manansingh96** [comment]
Fix for Issue 1 has to be done in PublicLab.Editor. I have done it and created a PR [here](https://github.com/publiclab/PublicLab.Editor/pull/301).

**Comment by gauravano** [comment]
Actually, we can transfer this issue to Publiclab.editor and solve all these issues in parts. @jywarren can you please transfer this issue to editor? 

I think @rexagod @IgorWilbert can help with issues 4 and 5. 

Thanks @skilfullycurled for such detailed issue :100:

**Comment by manansingh96** [comment]
@gauravano Yes. And I have found some more issues. I will create a post soon.

**Comment by gauravano** [comment]
Great @manansingh96! your post is welcome. Thanks!
